helm_deploy/values-test.yaml

```
@@ -12,7 +12,7 @@ generic-service:
     HMPPS_AUTH_BASE_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk"
     HMPPS_HANDOVER_BASE_URL: "https://arns-handover-service-test.hmpps.service.justice.gov.uk"
     OASYS_BASE_URL: "https://arns-oastub-test.hmpps.service.justice.gov.uk"
-    OASYS_RETURN_URLS: "https://arns-oastub-test.hmpps.service.justice.gov.uk,http://10.0.1,https://t2-b.oasys.service.justice.gov.uk,http://192.168.56.21:8080/ords"
+    OASYS_RETURN_URLS: "https://arns-oastub-test.hmpps.service.justice.gov.uk,http://10.0.1,https://t2-b.oasys.service.justice.gov.uk"
```

@neiltait try this:

```suggestion
    OASYS_RETURN_URLS: "https://arns-oastub-test.hmpps.service.justice.gov.uk,http://10.0.1,https://t2-b.oasys.service.justice.gov.uk,http://192.168.56.21"
```

Doesn't have to be the full return URL AFAIK

---

Wonder if it's the single forward / perhaps this is being interpreted as an escape char...